build-native.cmd

```diff
@@ -0,0 +1,107 @@
+::Build-native windows
+@echo off
+setlocal
+
+set BUILD_TYPE=Debug
+set CMAKE_BUILD_TYPE=Debug
+set HOST_OS=Linux
+
+set __binDir=%~dp0\bin
+set __srcDir=%~dp0\src\libuv
+set __CMakeBinDir=""
+set __IntermediatesDir=""
+set __BuildArch=x64
+set CMAKE_BUILD_TYPE=Debug
+set __TargetOS=Windows_NT
+set __binRuntimeDir=%~dp0\bin\%__TargetOS%.%__BuildArch%.%CMAKE_BUILD_TYPE%
+set __buildOutputDir=%~dp0\src\libuv\%CMAKE_BUILD_TYPE%
+:: All CI machines use this
+set GYP_MSVS_VERSION=2015
+
+:Arg_Loop
+:: Since the native build requires some configuration information before msbuild is called, we have to do some manual args parsing
+if [%1] == [] goto :ToolsVersion
+if /i [%1] == [Release]     ( set CMAKE_BUILD_TYPE=release&&shift&goto Arg_Loop)
+if /i [%1] == [Debug]       ( set CMAKE_BUILD_TYPE=debug&&shift&goto Arg_Loop)
+
+if /i [%1] == [AnyCPU]      ( set __BuildArch=x64&&set __VCBuildArch=x86_amd64&&shift&goto Arg_Loop)
+if /i [%1] == [x86]         ( set __BuildArch=x86&&set __VCBuildArch=x86&&shift&goto Arg_Loop)
+if /i [%1] == [arm]         ( set __BuildArch=arm&&set __VCBuildArch=x86_arm&&set __SDKVersion="-DCMAKE_SYSTEM_VERSION=10.0"&&shift&goto Arg_Loop)
+if /i [%1] == [x64]         ( set __BuildArch=x64&&set __VCBuildArch=x86_amd64&&shift&goto Arg_Loop)
```

Comment:
I have a small preference on not having two ways to specify amd64.  I would just stick with x64 and kill the other one.